index.html

```diff
@@ -291,6 +291,42 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
+    <footer>
+      <p class="footer-text">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </p>
+      <div class="contact-form-container">
+        <form action="https://formspree.io/f/mrgrjoyw" method="POST" class="contact-form">
+          <input
+            type="text"
+            class="contact-form-input"
+            name="name"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <input
+            type="text"
+            class="contact-form-input"
+            name="name"
+            placeholder="Email address"
+            required
+          />
+          <textarea
+            name="message"
+            id="form-message"
+            cols="30"
+            rows="10"
+            maxlength="500"
+            class="message"
+          >
```

Comment:
In order to be sure that the user will submit a message, it is important to mark as `required` the textarea field.

- [ ] Please ensure that textarea has the attribute of `required`.